CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
 Grammars:
 
+- enh(twig) update keywords list
```

Comment:
Please credit yourself here and on line ~30.... (as with other examples)

Reply:
Thanks for your review, it should be better with the last commit